src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -238,6 +238,21 @@ void InputProcessor::markObjectAsUsed(const std::string &objectType, const std::
     }
 }
 
+void cleanEPJSON(json &epjson)
+{
+    if (epjson.type() == json::value_t::object) {
+
+        epjson.erase("idf_order");
+        epjson.erase("idf_max_fields");
+        epjson.erase("idf_max_extensible_fields");
```

Comment:
This seems fine, I was worried about what happened if that key wasn't in the object, but I see from the docs that it will pass normally.